azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -235,8 +259,10 @@ def create_from_symmetric_key(cls, symmetric_key, hostname, device_id, **kwargs)
             arbitrary product info which is appended to the user agent string.
         :param proxy_options: Options for sending traffic through proxy servers.
         :type proxy_options: :class:`azure.iot.device.ProxyOptions`
+        :param int sastoken_ttl: The time to live (in seconds) for the created SasToken used for
+            authentication. Default is 3600 seconds (1 hour)
```

Comment:
random comment: I like to see the level of detail in our docstrings go up.  If you haven't looked at the public-facing docstrings in the other Azure SDKs, you should (I think I'm thinking of eventhub, but maybe uamqp).

Reply:
Yeah I agree, we could do a bit more. Once we get docgen occurring correctly, there's a number of improvements I think could be made.